lib/screens/profile_screens/recovery_phrase/interactor/viewmodels/recovery_phrase_bloc.dart

```diff
@@ -3,13 +3,20 @@ import 'dart:async';
 import 'package:bloc/bloc.dart';
 import 'package:seeds/screens/profile_screens/recovery_phrase/interactor/viewmodels/recovery_phrase_state.dart';
 import 'package:seeds/screens/profile_screens/security/interactor/viewmodels/bloc.dart';
+import 'package:seeds/utils/mnemonic_code/mnemonic_code.dart';
 
 /// --- BLOC
+const TWELVE_WORDS = 128;
 class RecoveryPhraseBloc extends Bloc<SecurityEvent, RecoveryPhraseState> {
-  RecoveryPhraseBloc() : super(RecoveryPhraseState.initial());
+  RecoveryPhraseBloc() : super(RecoveryPhraseState.initial(generateWords()));
 
```

Comment:
Why not do it direct in the contructor?

`: super(RecoveryPhraseState.initial(generateMnemonic(strength: TWELVE_WORDS).plit('-'))); `

So you save a function or even do it inside the factory `RecoveryPhraseState.initial()`.

Reply:
I would arguea that creating the function is more readale?

Reply:
> I still prefer lower cammel case for constants 👀

Is this a matter of preference or a Dart standard ?

Reply:
> Is this a matter of preference or a Dart standard ?

both.
https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names